guides/micronaut-java-records/micronaut-java-records.adoc

```diff
@@ -0,0 +1,193 @@
+include::{commondir}/common-header.adoc[]
+
+You are going to use https://docs.oracle.com/en/java/javase/16/language/records.html[Record Classes] in a Micronaut Application.
+
+____
+Record classes, which are a special kind of class, help to model plain data aggregates with less ceremony than normal classes.
+
+A record declaration specifies in a header a description of its contents; the appropriate accessors, constructor, equals, hashCode, and toString methods are created automatically. A record's fields are final because the class is intended to serve as a simple "data carrier".
+____
+
+include::{commondir}/common-requirements.adoc[]
```

Comment:
This includes:

`JDK 1.8 or greater installed with JAVA_HOME configured appropriately`

And that's not correct. You need JDK17 for this guide (well, it will work with 16 but we officially don't support it).

Reply:
I have added a `minJdk` placeholder which defaults to 1.8. Otherwise, it picks the guide min JDK